### DISCUSSION TITLE
Unable to transfer domain from Vercel to Cloudflare

### DISCUSSION THREAD

**Discussion by brandonweiss** [vercel/community#5357] [Help]
### Summary

I have a domain registered in Vercel. When I select “Transfer out” I get an auth code, but I can’t provide that auth code to Cloudflare because Cloudflare says “transfer conditions not met”, specifically the domain status is “server transfer prohibited”.

### Steps to Reproduce

N/A

**Comment by amyegan** [comment]
Hi Brandon! How long has it been since you originally registered the domain? [ICANN rules](https://www.icann.org/resources/pages/text-2012-02-25-en#:~:text=Please%20note%20that%20you%20may,60%20days%20after%20a%20transfer) require that a domain be registered for at least 60 days before it can be transferred to another registrar. That could be why the transfer is blocked.

https://vercel.com/docs/projects/domains/transfer-your-domain#transferring-domains-out-of-vercel

**Comment by brandonweiss** [comment]
Sorry, it turned out that Cloudflare doesn’t support `.one` domains. It just doesn’t explain that in the UI.
